Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_Malware_-_Incident_Enrichment.yml

```diff
@@ -0,0 +1,4565 @@
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "65":
+    id: "65"
+    taskid: e316fa9b-05b4-4ac3-8288-0337a4d40cb8
+    type: title
+    task:
+      id: e316fa9b-05b4-4ac3-8288-0337a4d40cb8
+      version: -1
+      name: '''Endpoint'' Context Data'
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "199"
+      - "200"
+      - "202"
+      - "203"
+      - "204"
+      - "205"
+      - "206"
+      - "207"
+      - "208"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 935,
+          "y": 3225
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 8504176f-fe3d-43b8-8b52-83607070b028
+    type: title
+    task:
+      id: 8504176f-fe3d-43b8-8b52-83607070b028
+      version: -1
+      name: '''IP'' Context Data'
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "81"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 930,
+          "y": 860
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "69":
+    id: "69"
+    taskid: 9d986bde-c7e1-42d7-809f-65ca58266e0e
+    type: title
+    task:
+      id: 9d986bde-c7e1-42d7-809f-65ca58266e0e
+      version: -1
+      name: '''Domain'' Context Data'
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "265"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 935,
+          "y": 3790
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "73":
+    id: "73"
+    taskid: cf407188-3109-45d4-86f1-d33c53353c13
+    type: condition
+    task:
+      id: cf407188-3109-45d4-86f1-d33c53353c13
+      version: -1
+      name: Verify Alerts
+      description: Verify that the ‘PaloAltoNetworksXDR.Incident.Alerts’ context key was extracted correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "116"
+      Verified:
+      - "113"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: PaloAltoNetworksXDR.Incident.alerts
+                accessor: '[0]'
```

Comment:
If that is an array, can we check if there are any alerts there? I prefer not to point to only 1 element in the array.

Reply:
changed to 'PaloAltoNetworksXDR.Incident.alerts' and used the 'Is not empty' condition.